gnd/src/androidTest/java/com/google/android/gnd/AcceptTermsOfServiceTest.java

```diff
@@ -33,11 +34,13 @@
 public class AcceptTermsOfServiceTest extends BaseMainActivityTest {
 
   @Inject FakeAuthenticationManager fakeAuthenticationManager;
+  @Inject FakeRemoteDataStore fakeRemoteDataStore;
 
   @Override
   public void setUp() {
     super.setUp();
-    fakeAuthenticationManager.setUser(FakeData.TEST_USER);
+    fakeAuthenticationManager.setUser(FakeData.USER);
+    fakeRemoteDataStore.setTestProject(FakeData.PROJECT);
```

Comment:
@shobhitagarwal1612  It appears that despite this call, the testProject in `FakeRemoteDataStore` is still `null`. Any idea what I could be doing wrong?

Reply:
If the project is used in the constructor of hilt injected classes, then the test project should be set before the setup() call. This points to another problem in the project where some business logic is present inside the constructors (e.g. feature sync and signin observser in MainViewModel)

Reply:
I tried to work around that but couldn't find an easy way, so I field #1045 and set default values in the fakes themselves for now.